src/main/nsis/samples/conf/process-conf.xml

```diff
@@ -63,7 +63,7 @@
                 <entry key="sfdc.debugMessagesFile" value="c:\dataloader\samples\status\sfdcSoapTrace.log"/>
                 <entry key="sfdc.endpoint" value="https://login.salesforce.com"/>
                 <entry key="sfdc.username" value="user@mycompany.com"/>
-                <!-- password specified below is invalid, please generate one using the encrypt.bat utility -->
+                <!-- password, specified below is invalid, please generate one entry in keystore only readable by yourself using the encrypt.bat utility -->
                 <entry key="sfdc.password" value="1111111111111111"/>
```

Comment:
Passwords "encrypted" without a key file will no longer be possible; might be good to make that clear. Alternatively, don't use the per-entry key file path at all; either always use the default secure path, or only set the path in one top-level place.

Reply:
Since we are giving each entry the option of username/password, we can let customer use differnt keyfile as well. Does not hurt us.